catt/controllers.py

```diff
@@ -211,10 +209,7 @@ def __init__(self, current_state, states, invert=False, fail=False):
         self._state_event = threading.Event()
         self._current_state = current_state
         if self._current_state in self._states_waited_for:
-            if fail:
-                raise ListenerError("Condition is already met (fail is set)")
```

Comment:
(unrelated to the PR)
I've removed this strange bit of code, as it isn't used anywhere. I honestly don't remember why I put it there.

Reply:
I feel you, I hate bits of code whose purpose I don't remember 😛 That's why I usually leave comments anywhere that's not completely obvious.

Reply:
I'm pretty sure it's never been used. I must have convinced myself that it would be useful at some point.